src/data-tutorials/users.tsx

```diff
@@ -1031,6 +1031,13 @@ const Users: User[] = [
     website: 'blog/tutorials/how-uniswap-works',
     tags: ['celosage', 'celo', 'beginner'],
   },
+  {
+    title: 'Optimizing Gas Consumption in Celo Smart Contracts: A Step-by-Step Guide',
```

Comment:
Hey can you remove ":" from title, it is creating issues while deploying

Reply:
Done